lib/msf/core/payload/stager.rb

```diff
@@ -200,8 +198,8 @@ def handle_connection(conn, opts={})
       # The connection should always have a peerhost (even if it's a
       # tunnel), but if it doesn't, erroring out here means losing the
       # session, so make sure it does, just to be safe.
-      if conn.respond_to? :peerhost
-        sending_msg << " to #{conn.peerhost}"
+      if conn.respond_to?(:peerhost)
+        #sending_msg << " to #{conn.peerhost}"
```

Comment:
This needs some discussion. I spoke to @wvu-r7 about it briefly, but even though line 201 results in `true`, accessing `conn.peerhost` results in a crash for named pipe transports. The socket type behind the scenes is a TCPSocket, not Rex socket type. Ping @hdm for help here please.